www/js/settings.json

```diff
@@ -92,8 +92,29 @@
               "long": "Long",
               "extralong": "Extra Long"
             }
+          },
+          "vishraam-options": {
+            "title": "Vishraam Options",
+            "options": {
+              "colored-words": "Colored Words",
+              "gradient-bg": "Gradient Background"
+            }
+          },
+          "vishraam-source": {
+            "title": "Vishraam Source",
+            "options": {
+              "sttm2": "STTM 2",
+              "igurbani": "iGurbani",
+              "sttm": "STTM Living" 
```

Comment:
Are we going to explain what these sources mean (where they come from)? some context in the help would be useful @maneetpaul

Reply:
also that they only show for certain themes.

Reply:
Yes! I will update the help to include the details for this